testplan/web_ui/testing/src/AssertionPane/LogGroup.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {StyleSheet, css} from "aphrodite";
+
+
+const LogGroup = (props) => {
+    if (props.logs && props.logs.length !== 0) {
+        const logInfos = [];
+        props.logs.forEach(element => {
```

Comment:
nitpick: it would be slightly neater to create this array in one step using `props.logs.map(...)`: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map